pkg/kubelet/cm/dra/manager.go

```diff
@@ -318,10 +327,9 @@ func (m *ManagerImpl) UnprepareResources(pod *v1.Pod) error {
 			continue
 		}
 
-		claimInfo := m.cache.get(*claimName, pod.Namespace)
-
+		claimInfo, ok := m.cache.get(*claimName, pod.Namespace)
 		// Skip calling NodeUnprepareResource if claim info is not cached
-		if claimInfo == nil {
+		if !ok {
 			continue
 		}
```

Comment:
This can be replaced with a `contains()` call

Reply:
claimInfo is actively used in the code below, so if we use `m.cache.contains` here, we'd have to call `m.cache.get` again. Is that what you're suggesting?